src/instances.ts

```diff
@@ -29,6 +29,9 @@ import { makeWatchRun } from './watch-run';
 
 const instances = {} as TSInstances;
 
+// TODO: workaround for issues with webpack typings
+type PatchedHookCallback = any;
```

Comment:
Bug #2 because of same same bug in #1 which makes the compiler property not match the webpack typings.